### ISSUE TITLE
Rajagopal model pelvis geometry

### ISSUE THREAD

**Issue by mrrezaie** [opensim-org/opensim-models#171]
Hi,

The pelvis geometries shipped with OpenSim v.4.5 are different from the geometries of the original Rajagopal model:

Rajagopal model pelvis:
https://github.com/opensim-org/opensim-models/blob/master/Models/Rajagopal/Geometry/r_pelvis.vtp
![image](https://github.com/opensim-org/opensim-models/assets/38867713/6713e59f-4fd6-478b-aebc-fbf340c8269f)

Rajagopal model pelvis with OpenSim v4.5 geometries:
https://github.com/opensim-org/opensim-models/blob/master/Geometry/r_pelvis.vtp
![image](https://github.com/opensim-org/opensim-models/assets/38867713/28ada65c-7d81-47eb-8734-250621f70d69)

As a user, I prefer to have all geometries in one place rather than separate _Geometry_ folders. Since the Rajagopal model (as well as Rajagopal_OpenSense) is now officially a part of OpenSim 4.5 models, it would be great if you copy the new pelvis geometries into the program Geometry folder. Both default and the new pelvis geometries can be kept and distinguished by different names, e.g. `r_pelvis_raj.vtp`.

Thank you in advance.

**Comment by aymanhab** [comment]
Indeed there're 5 different mesh files that have the same name but different contents that are used exclusively by the Rajagopal models. Ideally the solution is to unify the contents and adjust the model files rather than introduce another variant unless there's a compelling reason. @carmichaelong do you know the history of why these were introduced or what were the differences (resolution, CS, scaling,... ) Thank you. For reference these are
![Screenshot 2024-02-08 105526](https://github.com/opensim-org/opensim-models/assets/3972208/4d88a0a1-40f0-4055-b058-27314f8d2bc7)

**Comment by carmichaelong** [comment]
I don't know the exact differences or reasons for different versions of the geometry. However, in general, it often can make sense to have different files for models. For instance, a model could be slightly adjusted by changing muscle insertions and origins based on new data, and the previous .vtp file would lead to those points either being off the bone or inside of the bone mesh. Or, new bone data could be used so that muscle paths do not penetrate bone meshes.

In short, I'd be wary of combining .vtp's from different (and seemingly similar) models.

**Comment by aymanhab** [comment]
Thanks @carmichaelong I generally agree, however the diff (#lines) suggests that one file was obtained from the other by applying a transform, if that's the case then it'd be preferrable to use/apply the transform rather than keep duplicate mesh files around (when the transform can be made into the osim file). 
Any idea where we trace the .vtp files origin/history?

**Comment by aymanhab** [comment]
While the geometry looks similar there was no clear transform between the two meshes (vtp files). Considering that the issue of different meshes with same file name is not unique to this model I'd lean to leave this alone just make sure we distribute models that display correctly. Long term solution (where we're trending) is to zip the meshes with the model so that mesh file names are local to the model rather than global in a master list with unique names.
